lib/Target/AVR/AVRTargetMachine.cpp

```diff
@@ -43,7 +43,7 @@ AVRTargetMachine::AVRTargetMachine(const Target &T, const Triple &TT, StringRef
                                    CodeGenOpt::Level OL) :
   LLVMTargetMachine(T, "e-p:16:8:8-i8:8:8-i16:8:8-i32:8:8-i64:8:8-f32:8:8-f64:8:8-n8",
                     TT, ProcessCPU(CPU), FS, Options, RM, CM, OL),
-  SubTarget(TT, CPU, FS, *this)
+  SubTarget(TT, ProcessCPU(CPU), FS, *this)
```

Comment:
While you are at it, maybe rename `ProcessCPU(...)`? I had a look at what it does, and was kind of disappointed. Not much processing happening there. Maybe just `GetTargetCPU(...)`?